zbus/src/win32.rs

```diff
@@ -26,7 +26,6 @@ use winapi::{
     },
 };
 
-use crate::Address;
```

Comment:
* The commit log says you're deprecating it but you're in fact dropping it.
* I see potential for further splitting the changes.

Reply:
Not dropping, you can still use the legacy Address to parse or manipulate Address. But you can't use it to  connect anymore. I am fine with removing it now if you prefer.

Reply:
> I am fine with removing it now if you prefer.

Yes that's also (what I thought) we decided. I do not believe this new `DBusAddress` is a new API at all. We should just change `Address` to fit our needs. This is not an API typical users use, especially directly.

Reply:
I am not sure what you mean by "changing Address to fit our needs".. As I see it, DBusAddr is very different.. there are no simple way to "change" the exisiting code, honestly.

Reply:
Also addr could be a new zbus_addr crate. It's fairly standalone (only zbus::Error/Result is reused)

Reply:
Look, we already have a type to represent a dbus address: `Address`. This is good and we should continue with that. Now that type has a few issues and you already have solution for solving that and we can totally break API here. i-e merge DBusAddress into Address.

> Also addr could be a new zbus_addr crate.

Sure but we'll then not want the `Address` API. The user only needs 1 (main) type here and that's what they should have.